google-cloud-spanner/src/main/java/com/google/cloud/spanner/Struct.java

```diff
@@ -41,7 +43,9 @@
  * <p>{@code Struct} instances are immutable.
  */
 @Immutable
-public abstract class Struct extends AbstractStructReader {
+public abstract class Struct extends AbstractStructReader implements Serializable {
```

Comment:
How do we ensure that if we ever add another subclass of Struct, that remains serializable?